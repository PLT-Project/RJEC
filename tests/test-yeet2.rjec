struct lol {
    x int;
    y int;
    z bool; 
}

func foo (x struct lol) {
    y := struct lol {
        x : x.x,
        y : x.y + 1
    };

    printi(y.x);
    printi(y.y);
    printb(y.z);
}

func main() {
    yeet foo(struct lol {
        x : 1,
        y : 2
    });
}