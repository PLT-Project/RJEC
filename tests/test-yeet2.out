1
3
false